content/millennium/faq/common-issues.md

```diff
@@ -25,31 +25,36 @@ identical each time and if they are retrieving the same data set.
 
 ### Implement Paging
 
-To be compatible with a wide variety of FHIR services and for future-proofing,
+To be compatible with a wide variety of implementations and for future-proofing,
 we require that you implement a paging data retrieval scheme
-as defined by the [HL7 FHIR specification](https://www.hl7.org/fhir/dstu2/http.html#paging).
-Your app should handle paging for all resources, even if our FHIR server
+as defined by the [HL7 FHIR<sup>®</sup> specification](https://www.hl7.org/fhir/dstu2/http.html#paging).
+Your app should handle paging for all resources, even if our server
 does not support it yet. This ensures your app is compatible with more
-FHIR servers and for ours if we add paging in the future for
-performance.
+implementations (including future versions of our server if we add paging in the future for
+performance).
```

Comment:
These parentheses have a lot of 'future's

What about:
"This ensures your app is compatible with more implementations (including future versions of our server if we add paging to improve performance of resources that do not currently page)."

Or if it is too awkward to keep the idea that adding paging is to improve performance in the bottom sentence:
"Your app should handle paging for all resources, even if our server does not support it yet for a particular resource. This ensures your app is compatible with more implementations (including future versions of our server that may add pagination to resources your app consumes)."